src/main/java/Homework1/n_3.java

```diff
@@ -0,0 +1,28 @@
+package Homework1;
+
+import java.util.Scanner;
+
+public class n_3 {
+    public static void main(String[] args){
+
+        System.out.println("Введите пол:");
+
+        Scanner n = new Scanner(System.in);
+
+        String s = n.nextLine();
+
+        if(s.equals("b")){System.out.println("ну здорова, приятель");}
+
+        else
```

Comment:
it is wrong formatting, should be like next: 

```suggestion
       if (s.equals("b")) { 
          System.out.println("ну здорова, приятель");
       } else if (s.equals("g")) {
          System.out.println("привет,как дела?");
       }
```